.readthedocs.yml

```diff
@@ -32,7 +32,7 @@ build:
 # to build docs
 python:
   install:
-  - requirements: docs/requirements.txt
+  - requirements: requirements.txt
```

Comment:
Don't move this file. Docs deps are coupled with docs, not the JS side of the project.

Reply:
Just wanted to reduce the amount of potentially wasted time by pointing out things that are definitely highly undesired and break the initial structural assumptions.